package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "audion",
+  "version": "1.0.0",
+  "description": "Traces web audio API calls.",
+  "main": "index.js",
+  "engines": {
+    "node": "14"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.14.6",
+    "@babel/plugin-transform-modules-commonjs": "^7.14.5",
+    "@pixi/jsdoc-template": "^2.6.0",
+    "babel-jest": "^27.0.6",
+    "eslint": "^7.30.0",
+    "eslint-config-google": "^0.14.0",
+    "husky": ">=6",
+    "jest": "^27.0.6",
+    "jest-puppeteer": "^5.0.4",
+    "jsdoc": "^3.6.7",
+    "lint-staged": ">=10",
+    "mustache": "^4.2.0",
+    "pinst": ">=2",
+    "prettier": "^2.3.2",
+    "puppeteer": "^9.1.1",
+    "webpack": "^5.44.0",
+    "webpack-cli": "^4.7.2",
+    "yazl": "^2.5.1"
+  },
+  "scripts": {
+    "build:chrome-extension": "node src/build/make-chrome-extension.js",
+    "build:webpack": "webpack --mode production --config src/webpack.config.js",
+    "build": "npm run build:webpack && npm run build:chrome-extension",
+    "postinstall": "husky install",
+    "postpublish": "pinst --enable",
+    "prepublishOnly": "pinst --disable",
+    "test:integration:run": "JEST_PUPPETEER_CONFIG=test/.jest-puppeteer.config.json jest --config test/.jest.config.json",
+    "test:integration": "npm run build && npm run test:integration:run",
+    "test:jsdoc": "jsdoc -c .jsdoc.json",
+    "test:lint:eslint": "eslint src/**/*.js",
+    "test:lint:prettier": "prettier --check src/**/*.js",
+    "test:lint": "npm run test:lint:eslint && npm run test:lint:prettier",
+    "test:unit": "jest --config src/.jest.config.json",
+    "test": "npm run test:lint && npm run test:jsdoc && npm run test:unit && npm run test:integration"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/GoogleChrome/audion.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```

Comment:
Not sure if this is the correct license for projects in the GoogleChrome org. See this for an example:
https://github.com/GoogleChrome/omnitone/blob/master/LICENSE

Reply:
I'll change that. ISC is the `npm init` default and I saw that audion's main branch package.json license field is set to the same. https://github.com/GoogleChrome/audion/blob/main/package.json#L10 I didn't look at the LICENSE file though, which matches the repo you linked. https://github.com/GoogleChrome/audion/blob/main/LICENSE